nano/core_test/block_store.cpp

```diff
@@ -666,7 +666,10 @@ TEST (block_store, large_iteration)
 	{
 		auto transaction (store.tx_begin (true));
 		nano::account account;
-		nano::random_pool.GenerateBlock (account.bytes.data (), account.bytes.size ());
+		{
```

Comment:
Can we encapsulate this in a function so if we want to change it against we don’t have to change all call sites?

Reply:
Good idea to wrap these to prevent forgetting to lock as well. I wanted to make the mutex/cryptopp variables inaccessible so made them local to the source file which worked until I realised shuffle () was a template function, so I've made a static class instead where those members are private.